### DISCUSSION TITLE
Choosing models for faster inference

### DISCUSSION THREAD

**Discussion by arif334** [coqui-ai/TTS#956] [Qs for Live Q&A] [answered]
What are the best models (AM + Vocoder) for the fastest inference with a single-speaker LJspeech-like dataset? I have trained a few of the models. But it is not possible for me to train all possible combinations of the models due to resource limitations. 

If someone has already tried the various combinations of the models, it would be nice if they report their RTFs.

**Comment by skol101** [comment]
@arif334 I think this is related to https://github.com/coqui-ai/TTS/discussions/961 . I myself am currently more interested in vits vctk inference.

**Comment by erogol** [comment, marked as answer]
(answered in the call)
